doc/_static/mpl.css

```diff
@@ -277,3 +277,22 @@ div.twocol > div {
     display: inline-block;
     margin: 0 0.5em;
 }
+
+
+/* styling for version switcher */
+.navbar-brand {
```

Comment:
I like this styling, but still opposed to moving next to the logo for aesthetic reasons.